test/overlapped-checker/main.c

```diff
@@ -0,0 +1,23 @@
+#include <stdio.h>
+#include <stdlib.h>
+
+#include "../../deps/uv/include/uv.h"
+#include "../../deps/uv/src/uv-common.h"
+
+int main() {
```

Comment:
`int main(void) {` - `int main() {` is not a legal definition of main (it is in C++ but not in C.)

The function also needs a `return 0;` statement.